packages/client-benchmark/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@prisma/client-benchmark",
+  "version": "0.0.0",
+  "private": true,
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/prisma/prisma.git",
+    "directory": "packages/client-benchmark-tests"
+  },
+  "dependencies": {
+    "@aws-sdk/client-lambda": "^3.279.0",
+    "@opentelemetry/api": "1.4.1",
+    "@opentelemetry/core": "1.12.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.38.0",
+    "@opentelemetry/id-generator-aws-xray": "^1.1.1",
+    "@opentelemetry/instrumentation": "0.35.1",
+    "@opentelemetry/instrumentation-aws-lambda": "^0.35.1",
+    "@opentelemetry/propagator-aws-xray": "^1.2.0",
+    "@opentelemetry/resources": "1.12.0",
+    "@opentelemetry/sdk-trace-base": "1.12.0",
+    "@opentelemetry/sdk-trace-node": "1.12.0",
+    "@opentelemetry/semantic-conventions": "1.12.0",
+    "@types/node": "16.18.11",
+    "archiver": "5.3.1",
+    "arg": "5.0.2",
+    "aws-crt": "^1.15.14",
+    "csv-stringify": "^6.2.4",
+    "execa": "5.1.1",
+    "fast-glob": "3.2.12"
+  },
+  "scripts": {
+    "dev": "DEV=true node -r esbuild-register helpers/build.ts",
+    "build": "node -r esbuild-register helpers/build.ts",
+    "measure": "node dist/measureCli.js",
```

Comment:
Would it make sense to delay the GC for as much as we can here to avoid spikes in the data?